dpnp/linalg/dpnp_iface_linalg.py

```diff
@@ -40,7 +40,6 @@
 """
 
 # pylint: disable=invalid-name
-# pylint: disable=no-member
```

Comment:
Could you please remove that from `dpnp/dpnp_iface_statistics.py` also, because it's done globally in `.pre-commit-config.yaml`